x-pack/platform/plugins/shared/onechat/server/routes/mcp.ts

```diff
@@ -148,9 +148,9 @@ export function registerMCPRoutes({ router, getInternalServices, logger }: Route
         authz: { requiredPrivileges: [apiPrivileges.readOnechat] },
       },
       access: 'public',
-      summary: 'MCP server',
+      summary: 'Get MCP server (not supported)',
```

Comment:
@pgayvallet @joemcelroy I've reworded to try to make it clear that POST is the only one that works for MCP and it's basically for testing connectivity.

I've added verbiage to make it clear that `GET` and `DELETE` are basically meaningless. Not sure if we can just delete the `GET` and `DELETE` routes for MCP entirely here?